library/Fission/Internal/Fixture/Web.hs

```diff
@@ -0,0 +1,36 @@
+module Fission.Internal.Fixture.Web 
+  ( fissionURL
+  , failureResp
+  , failure502
+  , failure504
+  ) where
+
+import Fission.Prelude
+
+import Servant.Client
+import Servant.Client.Core.Request
+import Servant.Client.Internal.HttpClient
+
+import Network.HTTP.Types.Status
+import Network.HTTP.Types.Version
+import RIO.Seq as Seq
```

Comment:
Nitpick: I'd make this `import qualified`, since we're only using it that way. Prevents pain down the road with ambiguous imports. Typically importing with `as` will have a `qualified` unless you're re-exporting the module.